tests/test_quad/test_bayesian_quadrature.py

```diff
@@ -120,3 +146,10 @@ def test_integrate_wrong_input(bq, bq_no_policy, data):
         bq.integrate(fun=fun, nodes=wrong_nodes, fun_evals=fun_evals)
     with pytest.raises(ValueError):
         bq_no_policy.integrate(fun=None, nodes=wrong_nodes, fun_evals=fun_evals)
+
+
+@pytest.mark.parametrize("rng", [np.random.default_rng(42), 42])
+def test_integrate_runs_with_integer_rng(bq, data, rng):
+    # make sure integrate runs with both a rn generator and a seed.
```

Comment:
Again, I think we should avoid this pattern.

Reply:
removed since obsolete.